Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.messages.in

```diff
@@ -323,19 +323,19 @@ messages -> RemoteGraphicsContextGL NotRefCounted Stream {
     void GetInternalformativ(uint32_t target, uint32_t internalformat, uint32_t pname, size_t paramsSize) -> (std::span<const int32_t> params) Synchronous
     void SetDrawingBufferColorSpace(WebCore::DestinationColorSpace arg0)
 #if ENABLE(WEBXR)
-    [EnabledIf='webXREnabled()'] void CreateExternalImage(uint32_t externalImage, WebCore::GraphicsContextGL::ExternalImageSource arg0, uint32_t internalFormat, int32_t layer) NotStreamEncodable
-    [EnabledIf='webXREnabled()'] void DeleteExternalImage(uint32_t handle)
-    [EnabledIf='webXREnabled()'] void BindExternalImage(uint32_t target, uint32_t arg1)
-    [EnabledIf='webXREnabled()'] void CreateExternalSync(uint32_t externalSync, WebCore::GraphicsContextGL::ExternalSyncSource arg0) NotStreamEncodable
+    [EnabledIf='webXRPromptAccepted()'] void CreateExternalImage(uint32_t externalImage, WebCore::GraphicsContextGL::ExternalImageSource arg0, uint32_t internalFormat, int32_t layer) NotStreamEncodable
```

Comment:
these are the changes needed so we don't need to modify the generate-gpuwebgl script

Reply:
You need to make the changes to `RemoteGraphicsContextGLProxy.h` as I pointed out in https://github.com/WebKit/WebKit/pull/30612#discussion_r1671648125. Then you run `generate-gpup-webgl` script to update the automatically generated message handling. If you don't update `RemoteGraphicsContextGLProxy.h`, next time someone runs the script, the changes will be undone.

Reply:
Oh thank you, makes sense, I’ll change that. That is probably what happened to my changes last time